projects/aca-content/src/lib/components/sidenav/user-menu/user-menu.component.html

```diff
@@ -1,4 +1,5 @@
-<button mat-button class="aca-user-menu-button" [matMenuTriggerFor]="menu">
+<button mat-button class="aca-user-menu-button" [matMenuTriggerFor]="menu"
+title="{{'APP.TOOLTIPS.OPTIONS_SETTINGS' | translate}}">
```

Comment:
it's the user profile menu, but the toolltip is "Settings"

Reply:
Shane says there are more settings behind the avatar icon than just user profile settings. he said, "Options and settings is correct.